calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,168 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to a registry. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can run the registry scanner wherever there is a container runtime. And it doesn’t have to be for Kubernetes.
+
+To use registry scanner, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI.
+
+To deploy the registry scanner as a pod in Kubernetes cluster, we recommend that you define a Kubernetes job or cronjob.
+
+## Before you begin
+
+**Required**
+
+- Registry scanner is running where there is a container runtime
+- Valid registry credentials
+
+**Supported registry platforms**
+
+- Amazon Elastic Container Registry (ECR)
+- Azure Container Registry (ACR)
+- Google Container Registry (GCR)
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+- If you use the registry scanner with Docker, only tagged images are scanned. However, if you use the scanner with Amazon or Azure, all images (tagged and untagged) are scanned.
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Set up authentication to registry scanner](#set-up-authentication-to-registry-scanner)
+- [Set up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
```

Comment:
#scan-images-and-send-results-calico-cloud => #scan-images-and-send-results-to-calico-cloud

Reply:
Fixed.